Formula/semgrep.rb

```diff
@@ -160,6 +169,7 @@ def install
       cd "semgrep-core" do
         system "opam", "install", "--deps-only", "-y", "."
         system "opam", "exec", "--", "make", "all"
+        system "opam", "exec", "--", "make", "test"
```

Comment:
As I've mentioned, this really isn't the test we need here. See my previous comment for details. https://github.com/Homebrew/homebrew-core/issues/80551#issuecomment-878683071

Reply:
Ah sorry missed that. Makes sense will remove it from here